mocks/Client.go

```diff
@@ -58,6 +57,11 @@ func (_m *Client) CancelWorkflow(ctx context.Context, workflowID string, runID s
 	return r0
 }
 
+// Close provides a mock function with given fields:
```

Comment:
This comment seems incomplete?

Reply:
Oh, nevermind I guess these are autogenerated? Weird.

Reply:
Yes, `mockery` is rubbish and were I to have built this project I'd have never used it (callers should use the mocking library of their choice, we should not provide mocks IMO).

Reply:
Yeah, I totally agree with that philosophy